Path.cpp

```diff
@@ -32,31 +33,48 @@ Path::Path() {
 /* find
  * @param: none
  * Precondition: none
- * Postcondition:   a Path constructed from the PATH environment variable
- * return: Null
+ * Postcondition:   retunrs teh index of the directory containing program or  Path constructed from the PATH environment variable
```

Comment:
Typo: change "retunrs" to "returns" and "teh" to "the". You can just change this on your next commit